cmd/gpud/command/up.go

```diff
@@ -48,11 +48,6 @@ func cmdUp(cliContext *cli.Context) (retErr error) {
 		return err
 	}
 
-	if err := systemd.LogrotateInit(); err != nil {
```

Comment:
If we remove it, we should update gpud.service together? otherwise the file cannot be rotated.

Reply:
discussed offline, this is safe, once we roll out the lumberjack based logrotates